.circleci/config.yml

```diff
@@ -8,7 +8,19 @@ jobs:
         type: boolean
         default: True
     docker:
-      - image: circleci/clojure:openjdk-11-lein
+      - image: cimg/clojure:1.10.1
+        auth:
+          # from org-global context
+          username: $DOCKERHUB_USER
```

Comment:
Forgive my ignorance because it's been a while since I've made changes in Circle, but I assume these credentials are only available from the `main`/`master` branch and someone can't fork the project and echo these credentials out to steal?

Reply:
Yer they should be fine.

The circleci build won't run on forks unless it's explicitly disabled so contributors can't update the CI config to log out secrets, and they can't make a PR on the base project unless they have write permissions (only Signal AI devs at the moment).

Circle also tries to obfuscate secrets from the context/env vars in the build logs.